piksi_rtk_msgs/CMakeLists.txt

```diff
@@ -40,9 +41,12 @@ add_message_files(
     PosLlhMulti.msg
     ReceiverState.msg
     ReceiverState_V2_2_15.msg
+    ReceiverState_V2_3_15.msg
     TrackingState.msg
```

Comment:
Do you intentionally keep the old messages in the CMakeLists.txt (e.g., ReceiverState_V2_2_15.msg)

Reply:
Yep. Let's say you are playing an old bag with this specific message, then ros knows its definition and you don't have to checkout a specific commit and compile piksi_rtk_msgs folder just to play this bag.